#FemPosDeviation平滑相关参数
ref_weight_smooth: 1000000.0  #参考线平滑代价
ref_weight_path_length: 1.0  #参考线轨迹长度代价
ref_weight_ref_deviation: 2.5 #参考线偏移代价
# 二次规划几何相似度约束
x_lower_bound : -0.1
x_upper_bound : 0.1
y_lower_bound : -0.1
y_upper_bound : 0.1

#是否使用离散点平滑（Apollo）
use_discrete_smooth: false